test/apps/fetch.yml

```diff
@@ -1,4 +1,5 @@
 image: therobotcooperative/fetch
+distribution: melodic
```

Comment:
If you're getting the ROS version from the ROS_DISTRO environment in the container, does the information need to be duplicated here?

Reply:
This is to remember the ROS distribution associated with a given image so that we can make certain decisions without needing to launch a container.